projects/kit/components/multi-select/multi-select.template.html

```diff
@@ -52,8 +66,8 @@
                 tuiWrapper
                 appearance="icon"
                 automation-id="tui-multi-select__arrow"
-                tuiPreventDefault="mousedown"
```

Comment:
Why we no longer need to prevent it?

Reply:
It was preventing native dropdown opening, and seems like it's old code and I didn't find reason for this 🤔 but now I found another solution with `pointer-events:none' and can return preventing for "desktop-dropdown" mode (just in case)

Reply:
It is there so when you are focused on input and click on arrow, focus remains inside input and does not go away and immediately back, I believe.